config/src/main/java/org/springframework/security/config/annotation/web/configurers/ExpressionUrlAuthorizationConfigurer.java

```diff
@@ -108,6 +109,57 @@ public ExpressionInterceptUrlRegistry getRegistry() {
 			extends
 			ExpressionUrlAuthorizationConfigurer<H>.AbstractInterceptUrlRegistry<ExpressionInterceptUrlRegistry, AuthorizedUrl> {
 
+		private boolean isAnyRequestConfigured = false;
+
+		@Override
+		public AuthorizedUrl anyRequest() {
+			if (isAnyRequestConfigured) {
```

Comment:
Also, would you please prefix member variables with `this.`? That way it's easy to differentiate between member and local variables.

Reply:
Let's use `Assert.state(!this.anyRequestConfigured, "Can't configure anyRequest after itself")` sinces its a bit more compact

Reply:
> Let's use `Assert.state(!this.anyRequestConfigured, "Can't configure anyRequest after itself")` sinces its a bit more compact

Replaced IllegalStateException use by Assert.

Reply:
> Also, would you please prefix member variables with `this.`? That way it's easy to differentiate between member and local variables.

Migrated to follow this convention.